src/index_notation/transformations.cpp

```diff
@@ -1297,7 +1298,9 @@ std::ostream& operator<<(std::ostream& os,
   return os;
 }
 
-static bool compare(std::vector<IndexVar> vars1, std::vector<IndexVar> vars2) {
+namespace {
```

Comment:
Is there a need for the anonymous namespace?

Reply:
g++ was throwing a warning about `getRootParent` not being declared. I could just make it `static` like `compare`, but anonymous namespace appears to be a more C++-y way of doing it.